src/Setup.elm

```diff
@@ -50,44 +51,34 @@ init =
 
 stringToFloat : String -> Float
 stringToFloat number =
-    let
-        maybeFloat =
-            String.toFloat number
-    in
-    case maybeFloat of
-        Nothing ->
-            0
-
-        Just value ->
-            value
+    withDefault 0 <| String.toFloat number
 
 
 yearsInput : Int -> Input.Label Msg -> (Float -> Msg) -> Element Msg
 yearsInput value label onChange =
-    el [ Element.centerX, Element.width <| px 345 ] <|
-        Input.slider
-            [ Element.height (Element.px 20)
-            , Element.behindContent
-                (Element.el
+    { onChange = onChange
+    , label = label
+    , min = 2007
+    , max = 2019
+    , step = Just 1
+    , value = toFloat value
+    , thumb =
+        Input.defaultThumb
+    }
```

Comment:
Maybe give this record a name in a `let`?
Just my opinion to make it more readable and understandable.